components/monitoring/base/install_prometheus.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: appstudio-workload-monitoring-stack
+  namespace: appstudio-workload-monitoring
+spec:
+  targetNamespaces:
+  - appstudio-workload-monitoring
+
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: prometheus
+  namespace: appstudio-workload-monitoring
+spec:
+  channel: beta
+  installPlanApproval: Automatic
+  name: prometheus
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: prometheusoperator.0.56.3
```

Comment:
do not use startingCSV that will always install this one and will be updating to newest one. Without this line it will install newest one directly.

Reply:
make sense. done